### ISSUE TITLE
full datasets and model

### ISSUE THREAD

**Issue by ZIKO94ZIKO** [stormspirit03/Covid-mask-detection-using-tensorflow-object-detection-api#1]
Hi Sir , Could you please send me the datasets and your model , thanks un advance

**Comment by stormspirit03** [comment]
Hi
https://drive.google.com/folderview?id=1UAxVOv-7OF22AUTEbpG1Rj-pDVsbxF3M

**Comment by zongoalbert** [comment]
Sir i see just 51 photos , in your datasets , its possible to detect face and classify masks with camera cctv in street view or with drone view , many thanks .

**Comment by stormspirit03** [comment]
No, for that u need bigger dataset. I made it as fun project. All the
labeling has been done manually. Also accuracy is not that great. U can try
augmentation for more sample and try increasing neurons if have enough
computation power. Most of the project you work on flow will be same ..only
thing will change is dataset and  model as per requirement.
